modules/collectd/manifests/plugin/icinga.pp

```diff
@@ -0,0 +1,23 @@
+# == Class: collectd::plugin::file_handles
+#
+# Monitors the number of Icinga alerts
+#
+class collectd::plugin::icinga(
+) {
+
+  @file { '/usr/lib/collectd/icinga.sh':
+    ensure  => 'present',
+    mode    => '0755',
+    owner   => 'root',
+    group   => 'root',
+    content => file('collectd/usr/lib/collectd/icinga.sh'),
+    tag     => 'collectd::plugin',
+    require => Class['collectd::package'],
+  }
+
+  @collectd::plugin { 'icinga':
+    content => template('collectd/etc/collectd/conf.d/icinga.conf.erb'),
```

Comment:
This template doesn't look like it needs to be a template. It might be simpler to treat it as a file (assuming that is indeed simpler).